models/history-model.js

```diff
@@ -0,0 +1,14 @@
+const mongoose=require("mongoose");
+const {Schema}=mongoose;
+
+const historySchema=new Schema({
+  userId:{
+    type:mongoose.Schema.Types.ObjectId,
+    ref:"User"
+  },
+  videos:[{id:{type:mongoose.Schema.Types.ObjectId, ref:"Video"},active:Boolean}]
```

Comment:
You have already destructured mongoose object above . Hence, you can reafactor `tpye:Schema.Types.ObjectId`

Reply:
Yes